bigframes/session/__init__.py

```diff
@@ -405,6 +434,23 @@ def read_gbq_table(
     ) -> dataframe.DataFrame:
         """Turn a BigQuery table into a DataFrame.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.read_gbq_table("bigquery-public-data.ml_datasets.penguins")
```

Comment:
The first two lines are boilerplate that aren't really relevant to the code sample (but the import at least provides good context). Let's add a header describing this particular code sample:
```suggestion
            >>> bpd.options.display.progress_bar = None

        Read a whole table, with arbitrary ordering or ordering corresponding to the primary key(s).

            >>> df = bpd.read_gbq_table("bigquery-public-data.ml_datasets.penguins")
```